### DISCUSSION TITLE
2.3.100 -- information no longer processing?

### DISCUSSION THREAD

**Discussion by RagnarSTS** [Security-Onion-Solutions/securityonion#7079] [General] [answered]
I also have a completely empty kibana and such after the upgrade completed with no data points after the install completed.
No errors noted on the upgrade.
so-status shows everything started; though seems slower than usual to start things up.
Did the tweak mentioned in another thread despite not having capital letters - my test install is named "seco2"
https://github.com/Security-Onion-Solutions/securityonion/discussions/7073
After doing the update above the highstate completed without issue. Rather than restarting all individual docker containers just did a reboot...still nothing.
tcpdump definitely shows data still hitting the interface. Stenographer seems to still be making pcap files normally.
[soup.log](https://github.com/Security-Onion-Solutions/securityonion/files/7989763/soup.log)

Attaching soup.log as instructed.

**Comment by m0duspwnens** [comment]
Looking at the log, the hostname of the manager does appear to be `SECO2`, so the changes from the previous discussion would have been needed. Following the change to `ssl/init.sls`, you will need to run `salt \* state.apply ssl -l info` on the manager to apply the `ssl` state to all the nodes. Following this, it would be ok to reboot to force the docker containers to reload.

Once the manager has come back online, and running `so-status` on it shows everything as `OK` then run the following on the manager: `salt-call state.apply kibana.so_savedobjects_defaults -l info`.

**Comment by RagnarSTS** [comment, marked as answer]
Did everything twice just to make sure; no errors noted but still zero information processing through the stack.

**Reply by RagnarSTS** [reply]
Hotfix applied and rebooted.  Still nothing.  
I'll give another hour or so for ideas, then I'll just rebuild it fresh.  :)

**Reply by m0duspwnens** [reply]
On your searchnode and sensors, you can try to remove or move the relevant keys and certs (ca.crt, ca.key elasticsearch.crt, elasticsearch.key, influxdb.crt and influxdb.key, filebeat.crt, filebeat.key, filebeat.p8 ) under `/etc/pki`. Also, move `/etc/ssl/certs/intca.crt`. Once done, run `salt-call state.apply ssl -l info` on the node that the keys and certs were moved, then reboot the node to restart all the docker containers.

**Reply by RagnarSTS** [reply]
Still nothing processing.
Kind of fun to watch the certificates re-generate.
Nothing in elastic; kibana pops up with a million "no_shard_available_action_exception"

**Reply by m0duspwnens** [reply]
@RagnarSTS If you haven't rebuilt, the latest hotfix should resolve your issue. If you are still receiving the Kibana error regarding shards, you can run `salt-call state.apply kibana.so_savedobjects_defaults -l info` on your manager node once the hot fix has been applied and all other nodes have had a chance to run highstate.

**Reply by RagnarSTS** [reply]
Hotfix applied; state.apply completed, highstate completed, reboot completed ... still zero ingestion after about 30 minutes of letting it try to catch up.  
I think my biggest issue with the new docker world is the lack of simple logs in /var/log.  Life used to be simple.

**Reply by m0duspwnens** [reply]
Do you have just a standalone node? The logs are in `/opt/so/log` and it would be helpful to start with Filebeat, then Logstash, then Elasticsearch logs.

**Reply by RagnarSTS** [reply]
Aye it is a standalone test box that lasted much longer than anticipated.  Apologies I've already reformatted the drives and about 40% of the way through a new setup. 
One of these days we'll be ready for production.  :)
